dali/base/dadfs.cpp

```diff
@@ -4704,6 +4705,20 @@ protected: friend class CDistributedFilePart;
         else
             return false;
     }
+    virtual double getCost(const char * cluster) override
+    {
+        VStringBuffer path("storage/planes[@name='%s']", cluster);
+        Owned<IPropertyTree> plane = queryGlobalConfig().getPropTree(path);
+        if (!plane)
+            return 0.0;
+        double storageCostDaily = plane->getPropReal("cost/@storageAtRest", 0.0) * 12 / 365;
+
+        CDateTime dt;
+        getModificationTime(dt);
+        double fileAgeDays = difftime(time(nullptr), dt.getSimple())/(24*60*60);
+        double sizeGB = getDiskSize(true, true) / ((double)1024 * 1024 * 1024);
```

Comment:
think should be getDiskSize(true, false),  i.e. should use meta info without opening file parts if there.

Reply:
GB is that granular enough (probably), should be use MB, incase some exotic storage comes along that is super-expensive, but we can't tell if using .5GB of it?